packages/opencensus-instrumentation-mongodb/test/test-mongodb.ts

```
@@ -74,6 +74,10 @@ function assertSpan(
     expectedKind: SpanKind) {
   assert.strictEqual(rootSpanVerifier.endedRootSpans.length, 1);
   assert.strictEqual(rootSpanVerifier.endedRootSpans[0].spans.length, 1);
+  // @ts-ignore
```

What dos `@ts-ignore` do here?

---

Because `span.truncated` is a private field, but i need it to verify that the span are correctly ended. The easiest way was to just ignore the error.

---

OK. Could you add a comment to explain that?

---

Done